### DISCUSSION TITLE
我的学生包即将到期，但是无法再次验证

### DISCUSSION THREAD

**Discussion by Cx330-502** [community/community#54511] [GitHub Education]
### Hi everyone,

我最近收到消息如下：
> Your GitHub academic discount coupon will expire in 13 days
> + If you’re still eligible, reverify your academic affiliation.
> + If you’re no longer eligible, update your payment information to continue using GitHub Pro, or downgrade to GitHub Free.
> 
> If you have any questions, please contact GitHub Education.

并且需要我Reverify your academic affiliation，但是我在一个月前已经再次验证过了

现在显示：You were last verified as a student on Mar 2, 2023. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.

我不知道该怎么继续我的学生包

**Comment by Litetras** [comment]
same bro

**Comment by WangMinan** [comment]
same situation. have put an issue and raise a ticket days before but receive no reply

**Comment by Cx330-502** [comment]
As an update, I'm logged into GitHub today and there are no more messages prompting me to reverify your academic affiliation.

**Reply by WangMinan** [reply]
![image](https://github.com/community/community/assets/77133573/6ce0806b-25e6-48cd-8fe6-96d1b7526cb1)
谢谢你 我看了一下 应该是被修好了

**Comment by Lossita** [comment]


**Reply by WangMinan** [reply]
maybe just wait them to solve this or sent ticket to github with "contact github"

**Reply by Lossita** [reply]
no support staff reply my ticket only wish they fix it quickly:(

**Comment by priority3** [comment]
the same issue, but it's disappeared recently, and I have no `PRO` plan now, how can I resolve this?

**Comment by Cx330-502** [comment]
Maybe you need to wait for a few days

**Comment by Jaccelerator** [comment]
I have the same problem that when I click the button `Reverify your academic affiliation`, then input the table for **continue**, it prompts that `You were last verified as a student on . It is not necessary for you to re verify at this time.`  I do not know how to re-apply for github education.

**Reply by Yan-Daojiang** [reply]
If it just expired, maybe you should wait a while. I had the same problem before.

**Comment by Lvjinhong** [comment]
I am same，because 
![image](https://github.com/community/community/assets/96970081/3bbcbc94-9fdf-47b3-becd-93e18f70dd54)

but when I click,

![image](https://github.com/community/community/assets/96970081/14a4c71a-7f04-4015-9375-56b047cf8f04)

**Reply by WangMinan** [reply]
我就一直等着 然后他过了段时间就自己恢复了 感觉就是有这个bug

**Reply by petaleaf** [reply]
same trouble

**Reply by Kyriezhao11** [reply]
请问现在怎么样了

**Reply by Penguin-SAMA** [reply]
解决了吗

**Reply by ghost** [reply]
现在又有这个bug了 悲 我收到邮件了 续不了 提示下个月过期 难绷

**Reply by WangMinan** [reply]
我现在没有 挺正常的

**Reply by we-worker** [reply]
救命，我也是这个问题

**Reply by idealzouhu** [reply]
我也遇到了这个问题，续订按钮都没有

**Reply by Slimeding** [reply]
Yes, I am facing the same issue. Have you resolved it?

**Reply by TonyEinstein** [reply]
I have been stuck on this page for 7 days:  “We're still processing your application. Once this is complete you will have access to the student developer pack offers.”

**Comment by t4lhawi** [comment]
I think so

**Reply by Slimeding** [reply]
Yes, I am facing the same issue. Have you resolved it?

**Comment by TonyEinstein** [comment]
Me, too. Here's my question discussion link:  [https://github.com/orgs/community/discussions/150725](https://github.com/orgs/community/discussions/150725)
